src/test/kotlin/com/beust/klaxon/KlaxonTest.kt

```diff
@@ -55,7 +55,61 @@ class KlaxonTest {
             )
         }
 
-        assertEquals("""{ "1" : null , "2" : 2 } """, j.toJsonString())
+        assertEquals("""{"1":null,"2":2}""", j.toJsonString())
```

Comment:
in pretty-print version there will be spaces but if no pretty-print enabled then it should be as compact as possible